lusSTR/tests/test_filters.py

```diff
@@ -183,7 +183,7 @@ def test_EFMoutput_format(outputdir, datatype, software, tmp_path):
         "--str-type",
         datatype,
         "--input",
-        "WD",
+        str(inputfile),
```

Comment:
I don't understand the significance of this change throughout this test script.

Reply:
It was erring out with just using the ```WD```. Really, the input doesn't matter because it's skipping the initial ```convert``` step anyway for all of these tests.